tools/run-container

```diff
@@ -360,18 +367,20 @@ wait_for_boot() {
     fi
 }
 
-start_container() {
-    local src="$1" name="$2"
+start_instance() {
+    local src="$1" name="$2" use_vm="$3"
     debug 1 "starting container $name from '$src'"
-    lxc launch "$src" "$name" || {
+    launch_flags=()
```

Comment:
This lays foundation for another change I have in mind: use ephemeral containers.